requirements.txt

```diff
@@ -1,6 +1,8 @@
 pbr>=1.8
 netaddr>=0.7.18
-requests>=2.9.1
+requests
```

Comment:
Can you explain the rationale behind changing version requirements of "requests"?